pkg/virt-controller/watch/application.go

```diff
@@ -74,6 +75,20 @@ const (
 
 var (
 	containerDiskDir = filepath.Join(util.VirtShareDir, "/container-disks")
+
+	leaderGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "leading_virt_controller",
+			Help: "Indication for an operating virt-controller.",
+		},
+	)
+
+	readyGauge = prometheus.NewGauge(
+		prometheus.GaugeOpts{
+			Name: "ready_virt_controller",
+			Help: "Indication for a virt-controller that is ready to take the lead.",
```

Comment:
Ahh, you're talking about something different.
I think that the `leader` is still `ready`...
Otherwise, it will make the alert complex: in case you have just one controller running, we'll have to check that there is at least 1 ready to not fire the alert.